content/posts/2022-08-17-ocp-on-m1.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Running ocp on M1 machines"
+date: 2022-08-17T08:58:10+05:30
+---
+
+From the 2.4.1 release, CRC has been running on macOS aarch64 M1 CPUs.
+However, the support was limited to podman preset on M1. We've been asked numerous times when CRC openshift
+preset would get M1 support. This has changed with the recently released [crc 2.7.1](https://github.com/code-ready/crc/releases/tag/v2.7.1).
+This blog post will describe the challenges we had to overcome when we decided to add that support.
+
+# Background
+
+CRC is a tool which allows you to run an OpenShift cluster or podman containers on a laptop or local workstation.
+It can be used on a macOS, Windows or Linux machine.
+It runs its OpenShift cluster/podman runtime by starting a virtual machine (VM) using a disk image built in advance and packaged in what we call a _bundle_.
+On each platform, we try to use native hypervisors.
```

Comment:
, such as KVM on Linux, Hyper-V on Windows, and using the virtualization framework on macOS.